operator/pkg/controller-runtime/cell.go

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package controllerruntime
```

Comment:
is the mismatch between directory and pkg name intentional?

Reply:
yes it's somehow intentional. It's more or less the same way in the library itself -> https://github.com/kubernetes-sigs/controller-runtime/blob/main/alias.go#L17